### PR TITLE
IMAP synchronization  based on imapsync

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/6211

### CODE REVIEW

File: administrator-manual/en/mail.rst [hunk 1]
```diff
@@ -360,6 +360,34 @@ some restrictions. It could check:
...
+.. _email_imap_synchronization:
+
+IMAP synchronization
+====================
+
+IMAP synchronization is based on an IMAP transfer tool called Imapsync.
+The purpose is to migrate an IMAP account emais from a remote server to the account
+of your email server. The migration is recursive and incremental,
+you can run it as many times you want, the emails will be copied locally
+if they do not exist on the local server.
```
Comment (DavidePrincipi) [inline administrator-manual/en/mail.rst:372]:
```suggestion
of your email server. 

The migration is recursive and incremental and 
can be repeated as many times as needed. The emails will be copied locally
if they do not exist on the local server.
```

File: administrator-manual/en/mail.rst [hunk 2]
```diff
@@ -360,6 +360,34 @@ some restrictions. It could check:
...
+
+The system administrator of the local |product| does not need to know the
+password of the local user, the IMAP admin authentication is used. However, he
+has to know the password of the remote IMAP account, except if the IMAP admin
+authentication is implemented also for this email server.
+
+If you plan to copy email from a |product| to another |product|,
+the IMAP admin user is the account ``vmail``. The password can be found at
+:file:`/var/lib/nethserver/secrets/vmail`.
```
Comment (DavidePrincipi) [inline administrator-manual/en/mail.rst:381]:
```suggestion
If the remote IMAP server is also a |product|,
the IMAP admin user is ``vmail`` and its password can be read from
:file:`/var/lib/nethserver/secrets/vmail`.
```

File: administrator-manual/en/mail.rst [hunk 3]
```diff
@@ -360,6 +360,34 @@ some restrictions. It could check:
...
+====================
+
+IMAP synchronization is based on an IMAP transfer tool called Imapsync.
+The purpose is to migrate an IMAP account emais from a remote server to the account
+of your email server. The migration is recursive and incremental,
+you can run it as many times you want, the emails will be copied locally
+if they do not exist on the local server.
+
+The system administrator of the local |product| does not need to know the
+password of the local user, the IMAP admin authentication is used. However, he
```
Comment (DavidePrincipi) [inline administrator-manual/en/mail.rst:375]:
```suggestion
password of the local user. However, he
```

File: administrator-manual/en/mail.rst [hunk 4]
```diff
@@ -360,6 +360,34 @@ some restrictions. It could check:
...
+has to know the password of the remote IMAP account, except if the IMAP admin
+authentication is implemented also for this email server.
+
+If you plan to copy email from a |product| to another |product|,
+the IMAP admin user is the account ``vmail``. The password can be found at
+:file:`/var/lib/nethserver/secrets/vmail`.
+
+In the IMAP synchronization panel you must use a specific username
+``username@domain.com*vmail`` and the ``vmail`` password.
```
Comment (DavidePrincipi) [inline administrator-manual/en/mail.rst:384]:
```suggestion
The username with a ``*vmail`` suffix (e.g. ``username@domain.com*vmail``) and the ``vmail`` password has to be set in the IMAP synchronization panel.
```

File: administrator-manual/en/mail.rst [hunk 5]
```diff
@@ -360,6 +360,34 @@ some restrictions. It could check:
...
+
+.. note:: The developer of Imapsync has written a page about servers supporting
+         the IMAP admin authentication, it is available `on his website 
+         <https://imapsync.lamiral.info/FAQ.d/FAQ.Admin_Authentication.txt>`_
```
Comment (DavidePrincipi) [inline administrator-manual/en/mail.rst:388]:
```suggestion
.. note:: 

    List of `IMAP servers with admin authentication <https://imapsync.lamiral.info/FAQ.d/FAQ.Admin_Authentication.txt>`_ in Imapsync documentation
```

File: administrator-manual/en/mail.rst [hunk 6]
```diff
@@ -360,6 +360,34 @@ some restrictions. It could check:
...
    pair: email; filter
 
+.. _email_imap_synchronization:
+
+IMAP synchronization
+====================
+
+IMAP synchronization is based on an IMAP transfer tool called Imapsync.
+The purpose is to migrate an IMAP account emais from a remote server to the account
+of your email server.
```
Comment (DavidePrincipi) [inline administrator-manual/en/mail.rst:370]:
```suggestion
The purpose is to migrate email messages from a remote IMAP account to a
local one.
```

File: administrator-manual/en/mail.rst [hunk 7]
```diff
@@ -360,6 +360,34 @@ some restrictions. It could check:
...
+IMAP synchronization is based on an IMAP transfer tool called Imapsync.
+The purpose is to migrate email messages from a remote IMAP account to a
+local one.
+
+The migration is recursive and incremental and 
+can be repeated as many times as needed. The emails will be copied locally
+if they do not exist on the local server.
+
+The system administrator of the local |product| does not need to know the
+password of the local user. However, he
```
Comment (gsanchietti) [inline administrator-manual/en/mail.rst:377]: Try to avoid male-only pronouns:

```suggestion
password of the local user. However, administrator
```

**Pull request merged**